packages/genesys-spark-components/src/components/stable/gux-popover/gux-popover.tsx

```diff
@@ -114,11 +115,14 @@ export class GuxPopover {
   }
 
   private updatePosition(): void {
-    const forElement = findElementById(this.root, this.for);
     // This is 13 because this makes the arrow look aligned
     const arrowLen = 13;
 
-    if (this.popupElement) {
+    if (
+      this.popupElement &&
+      getContainingDocument(this.root).getElementById(this.for)
```

Comment:
If I am understanding the purpose of the `findElementById` utility, won't this fail if the target element is in a shadow DOM? Is there a reason to not just check if `forElement` exists instead?